src/java.base/share/classes/java/text/DigitList.java

```diff
@@ -394,6 +401,19 @@ private void set(boolean isNegative, String s,
 
     /**
      * Round the representation to the given number of digits.
+     * This method is reserved for Long and BigInteger representations.
+     * @param maximumDigits The maximum number of digits to be shown.
+     *
+     * Upon return, count will be less than or equal to maximumDigits.
+     */
+    private void round(int maximumDigits) {
```

Comment:
If the method is only used for `Long` and `BigInteger`, probably use the specific name instead of explaining it in the description and overloading would be more clear to me.